tests/FSharp.Compiler.ComponentTests/Miscellaneous/MigratedTypeCheckTests.fs

```diff
@@ -63,7 +63,7 @@ let ``type check neg14`` () = singleNegTest ( "typecheck/sigs") "neg14"
 [<FactForDESKTOP>]
 let ``type check neg15`` () = singleNegTest ( "typecheck/sigs") "neg15"
 
-[<FactForDESKTOP>]
+[<FactForDESKTOP(Skip = "Failing in new test framework")>]
```

Comment:
It shouldn't be failing, yes, it covers a lot of active patterns and shouldn't be just disabled.

Reply:
Not sure why I `Skipped` this tests. Just enabled again.

Reply:
> It shouldn't be failing, yes, it covers a lot of active patterns and shouldn't be just disabled. Would also be curious to see what s the exact failure on desktop.

@vzarytovskii @psfinaki So [this](https://dev.azure.com/dnceng-public/public/_build/results?buildId=795626&view=logs&j=09cc464c-89cd-5758-006d-ba9e3958cfa9&t=098678ff-b2f3-5c88-44d3-4c7be2f47e75) is the test that was failing. To make sure Im covering this on Linux and mac I moved `neg16.fs` content to separate test.

It is hard to understand  why and where exactly fails TBH, if you know please let me know.

```fsharp
  Failed Miscellaneous.FsharpSuiteMigrated_TypeCheckTests.type check neg16 [181 ms]
  Error Message:
   NUnit.Framework.AssertionException :   D:\a\_work\1\s\tests\fsharp\typecheck/sigs\neg16.bsl
  Expected: <empty>
  But was:  "Unexpectedly missing (expected, not actual):
        ''a option'
        'Choice<'a,'b,'c> option'
        'Choice<'a,'b> option'
        'Choice<'a,'b>' 
        'string'

Unexpectedly present (actual, not expected):
        'Choice<'a,'b,'c> option'    
        'Choice<'a,'b> option'    
"
```

Reply:
Hm, odd, and it's only on full framework...

Reply:
I guess eventually it's a similar thing with some non-updated error range in neg16.bsl, which you can investigate in the same manner as other tests we've looked into.

Why only on Windows - I guess because we only run it on Windows (why this - is a different topic 👀)

I can look into this tomorrow also I think.

Reply:
Thanks, that would be awesome.